viewer/src/logic/recent.spec.ts

```diff
@@ -1,64 +1,47 @@
-import { LogEntry, PlayerEnum } from "@gaia-project/engine";
+import Engine, { LogEntry, PlayerEnum } from "@gaia-project/engine";
 import { expect } from "chai";
-import { recentMoves } from "./recent";
+import { recentMoves, roundMoves } from "./recent";
 
 describe("Moves", () => {
-  describe("recentMoves", () => {
+  describe("recentMoves and roundMoves", () => {
     const tests: {
       name: string;
-      give: { moveHistory: string[]; logEntries: LogEntry[] };
-      want: string[];
+      give: { moveHistory: string[] };
+      want: { recentMoves: string[]; roundMoves: string[] };
     }[] = [
       {
         name: "player starts",
         give: {
           moveHistory: [
             "init 2 randomSeed2",
-            "p2 rotate",
```

Comment:
this is removed, but below it's in `want.roundMoves`

Reply:
the PR is not ready - just wanted to get the viewer tests to run..